src/renderer/modals/VoteTron/fields/VotesField.js

```diff
@@ -129,19 +131,26 @@ const SubTitle = styled(Box).attrs(() => ({
   color: ${p => p.theme.colors.palette.text.shade60};
 `;
 
+const RightFloating = styled.div`
+  position: absolute;
+  right: 0;
+  padding: 8px;
+  pointer-events none;
+  opacity: 0;
+  transition: opacity 200ms ease-out;
```

Comment:
what is the transition doing? this is beyond fixing the bug no? We intentionally removed some that existed in the past in pixel push because it didn't feel right. when you focus something you want to instantly get the thing "focused", animation was kinda noising the focus. (happy to discuss more & ask a designer)